serverless_sim/src/mechanism.rs

```diff
@@ -136,7 +136,7 @@ impl ConfigNewMec for Config {
                 return None;
             }
             "scale_sche_joint" => {
-                let allow_sche = vec!["pos"];
+                let allow_sche = vec!["pos", "consistenthash"];
```

Comment:
Seems current sche type is "no scaler" because you just select one node from all of the node, just move to "no scaler"